package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "react-aria-menubutton",
-  "version": "4.3.1",
+  "version": "4.4.0",
```

Comment:
Let's remove this from the PR. This is a breaking change, I think, not a patch. But I'd prefer not to change versions in the PR.